db/large_data_handler.cc

```diff
@@ -120,12 +120,21 @@ cql_table_large_data_handler::cql_table_large_data_handler(gms::feature_service&
     , _record_large_cells([this] (const sstables::sstable& sst, const sstables::key& pk, const clustering_key_prefix* ck, const column_definition& cdef, uint64_t cell_size, uint64_t collection_elements) {
         return internal_record_large_cells(sst, pk, ck, cdef, cell_size, collection_elements);
     })
-    , _feat_listener(_feat.large_collection_detection.when_enabled([this] {
+    , _record_large_partitions([this] (const sstables::sstable& sst, const sstables::key& pk, uint64_t partition_size, uint64_t rows, uint64_t range_tombstones, uint64_t dead_rows) {
+        return internal_record_large_partitions(sst, pk, partition_size, rows);
+    })
```

Comment:
A few sentences in the commitlog description, describing what is happening here, would have made the review easier.

Reply:
I added new a description.